integration/js/VideoEnd2EndTest.js

```diff
@@ -0,0 +1,74 @@
+const {OpenAppStep, WaitForMeetingToBeCreated, JoinVideoTestMeetingStep, ClickStartLocalVideoButton, ClickStopLocalVideoButton, ClickHasStartedLocalVideoTileButton, ClickAddVideoTileButton, ClickGetLocalVideoTileButton, ClickHaveVideoTileForAttendeeIdButton, ClickGetAllVideoTilesButton, ClickGetAllRemoteVideoTilesButton, ClickHaveVideoTilesWithStreamsButton, ClickRemoveAllVideoTilesButton, WaitForAllParticipantsToJoinMeeting, WaitForAllParticipantsToTurnVideoOn, WaitForAllParticipantsToTurnVideoOff} = require('./steps');
+const {TestUtils} = require('kite-common');
+const SdkBaseTest = require('./utils/SdkBaseTest');
+const {TileStateCheck, AddVideoTileCheck, GetAllVideoTilesCheck, HasStartedLocalVideoTileCheck, HaveVideoTilesWithStreamsCheck, GetAllRemoteVideoTilesCheck, GetLocalVideoTileCheck, HaveVideoTileForAttendeeIdCheck} = require('./checks');
+const { v4: uuidv4 } = require('uuid');
+
+class VideoTestEnd2End extends SdkBaseTest {
+  constructor(name, kiteConfig) {
+    super(name, kiteConfig, "TestAppVideoEnd2End");
+  }
+
+  async runIntegrationTest() {
+    const attendeeName = uuidv4();
+    const session = this.seleniumSessions[0];
+    const useSimulcast = this.useSimulcast;
+    await WaitForMeetingToBeCreated.executeStep(this, session);
+    await OpenAppStep.executeStep(this, session);
+    await JoinVideoTestMeetingStep.executeStep(this, session, attendeeName, this.meetingTitle, useSimulcast);
```

Comment:
I think you can just create the attendeeName inside this step, unless you are using it somewhere else.

Reply:
You are right, I don't need to create the attendeeName by myself. Will remove it.